src/Assets/LightSkyDefense/Scripts/SpawnCreditOnDestruction.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpawnCreditOnDestruction : MonoBehaviour
+{
+    public Credit Credit;
+    public int Value;
+
+    private bool isQuitting = false;
```

Comment:
private variables should start with an underscore according to the Microsofts naming convention.

```suggestion
    private bool _isQuitting = false;
```